### ISSUE TITLE
Questions for the documentation of the project

### ISSUE THREAD

**Issue by Sung-HwanLee** [snu-sf-class/swpp202201#37]
Please ask questions about the project documentation by replying to this issue.

**Comment by 2wind** [comment]
Requirement and Specification이나 Planning, Progress report 같은 제출해야 하는 문서들은 팀 단위로 보내는 것이 맞나요? 즉 팀 단위로 작성해서, 한 명이 조교님께 메일로 발송하면 될까요? 일부 문서에 대해 Each team should submit... 대신 You should submit... 식으로 서술된 부분이 있어 확인차 질문드립니다.

**Comment by strikef** [comment]
You're right. All documentations are to be submitted on a per team basis. You should work on the documents together, but each of them should be submitted only once per team.

**Comment by cdh981009** [comment]
project pdf에 다음과 같이 적혀져 있는 부분에 대해 질문이 있습니다.
```
* After each commit, the project should work correctly. TA will check
cmake -Bbuild
cmake --build build --target swpp-compiler
cd build && ctest
from the root directory of the branch.
```
CI 설정을 하려고 알아보니 아마 ctest가 interpreter나 alive2를 이용하는 .sh같은것을 실행해야 할 것 같은데, alive2나 interpreter가 깔려있는 위치에 따라서 ctest가 동작할 수 도 있고 안할 수 도 있지 않나요?

**Comment by cdh981009** [comment]
I have questions about requirement and specification document.

Since we have to include 3 IR examples for each optimization, number of pages adds up pretty quickly. Is 7 page restriction for sprint 1 (and 5 page for other sprints) a fixed thing? (Or maybe does this imply that maybe our team is planning too many different types of optimizations at once? )

Also related to this, do we also have to include 3 IR examples for existing llvm passes?

Thank you. (also can you answer to the question above)

**Comment by strikef** [comment]
@cdh981009

We'll check if your test works correctly on the CI, and 'try to' run your tests as much as possible even if they depend on programs or files in different directory. This is not mandatory, but it would be a good practice to factor out such external directories into CMake variables. Doing so will make your test script more robust, open to change, and ease our grading ;)

The page limit is a hard limit, and you should also include IR examples for existing LLVM passes as well.
